src/industry.h

```diff
@@ -60,20 +60,38 @@ DECLARE_ENUM_AS_BIT_SET(IndustryControlFlags);
  * Defines the internal data of a functional industry.
  */
 struct Industry : IndustryPool::PoolItem<&_industry_pool> {
+	struct ProducedHistory {
+		uint16_t production; ///< Total produced
+		uint16_t transported; ///< Total transported
+
+		uint8_t PctTransported() const
+		{
+			if (this->production == 0) return 0;
+			return  ClampTo<uint8_t>(this->transported * 256 / this->production);
```

Comment:
```suggestion
			return ClampTo<uint8_t>(this->transported * 256 / this->production);
```
Remove one of the double spaces.